custom_object.rb

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env ruby
+
+# This class is based off of a simple oxy/prop "lampworking" torch.
+class GlassBlowingTorch
+  def initialize
+    @oxygen = 0
+    puts 'Your oxygen is disconnected.'
+    @propane = 0
+    puts 'Your propane is disconnected.'
+    @flame_temp = 0
+    puts 'Currently, there is no flame.'
+  end
+
+  def instructions
+    puts 'Basic instructions:'
+    puts '1) Connect the propane tank to the torch gas line.'
+    puts '2) Turn the propane dial to 1.'
+    puts '3) Ignite flame.'
+    puts '4) Connect oxygen tank.'
+    puts '5) Experiment with turning the oxy/prop dials'
+    puts '6) Do not go over 2 turns on either dial.'
+    puts '7) you can only turn the flame up in temp,'
+    puts 'you must switch off and restart.'
+  end
+
+  def connect_propane(connect)
+    while connect == true
+      @propane = 1
+      puts 'You just connected the propane tank.'
+      break
+    end
+  end
+
+  def connect_oxygen(connect)
+    while connect == true
+      @oxygen = 1
+      puts 'You just connected the oxygen tank.'
+      break
+    end
+  end
+
+  def propane_dial(turn)
```

Comment:
`level`?  `dial_setting`?  `turn` is not really descriptive of what you're setting the propane dial to...